zap/src/main/java/org/parosproxy/paros/core/scanner/ScannerParam.java

```diff
@@ -141,6 +142,14 @@ public class ScannerParam extends AbstractParam {
      */
     static final String ENCODE_COOKIE_VALUES = ACTIVE_SCAN_BASE_KEY + ".encodeCookieValues";
 
+    /**
+     * Configuration key to write/read the {@code scanBodyPlain} flag.
+     *
+     * @since 2.17.0
+     * @see #scanPlainBody
+     */
+    static final String SCAN_PLAIN_BODY_PARAM = ACTIVE_SCAN_BASE_KEY + ".scanPlainBody";
+
```

Comment:
This should be removed.

Reply:
Right I realized later I was handled differently